hadoop-hdds/docs/content/security/SecureOzone.zh.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "安全化 Ozone"
+date: "2019-April-03"
+summary: 简要介绍 Ozone 中的安全概念以及安全化 OM 和 SCM 的步骤。
+weight: 1
+icon: tower
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+
+# Kerberos
+
+Ozone 集群的安全依赖于 [Kerberos](https://web.mit.edu/kerberos/)。过去 HDFS 支持在隔离的安全网络中运行，因此可以不进行安全化的集群部署。
+
+Ozone 在这方面与 HDFS 保持一致，但不久之后将 _默认启用安全机制_ 。目前，Ozone 集群启用安全机制需要将配置 **ozone.security.enabled** 设置为 _true_ ，以及将 **hadoop.security.authentication** 设置为 _kerberos_ 。
+
+参数 | 值
+----------------------|---------
+ozone.security.enabled| _true_
+hadoop.security.authentication| _kerberos_
+
+# Tokens #
+
+Ozone 使用 token 的方法来防止 Kerberos 服务器负载过重，当每秒处理上千个请求时，Kerberos 可能无法很好地工作。所以，每次当用户完成一次认证之后，Ozone 会向用户颁发代理 token 和块 token，应用程序可以使用这些 token 来对集群进行特定的操作，就像它们持有 kerberos 凭据一样，Ozone 支持以下类型的 token。
+
+### 代理 Token ###
+代理 token 允许应用模拟用户的 kerberos 凭据，它基于 kerberos 的身份认证，由 OM 颁发，当集群启用安全机制时，代理 token 功能默认启用。
+
+### 块 Token ###
+
+用户通过块 token 来读写一个块，它的作用是让数据节点知道用户是否有对块进行读和修改的权限。
+
+### S3 Token ###
```

Comment:
In [doc of 0.5.0-beta](https://hadoop.apache.org/ozone/docs/0.5.0-beta/security/secureozone.html), the S3 Token has been changed to S3AuthInfo, could we make it the same here?